features/mask/distsql/handler/src/main/java/org/apache/shardingsphere/mask/distsql/handler/update/DropMaskRuleStatementUpdater.java

```diff
@@ -41,6 +41,9 @@ public void checkSQLStatement(final ShardingSphereDatabase database, final DropM
     }
     
     private void checkToBeDroppedMaskTableNames(final String databaseName, final DropMaskRuleStatement sqlStatement, final MaskRuleConfiguration currentRuleConfig) {
+        if (sqlStatement.isIfExists()) {
+            return;
+        }
```

Comment:
If the isIfExists is true, we should process not existed rule in `updateCurrentRuleConfiguration `.